src/components/Trending.js

```diff
@@ -19,7 +19,7 @@ export default function Trending({ refresh, setRefresh }) {
     promise.then((res) => {
       setHashtags(res.data);
     });
-  }, [refresh, config]);
+  }, [refresh, hashtags.length]);
```

Comment:
Maintain config in the dependencies array, because is a paramater for listTrending function.